code/game/objects/items/devices/magnetic_lock.dm

```diff
@@ -11,46 +11,51 @@
 	icon = 'icons/obj/magnetic_locks/centcom.dmi'
 	icon_state = "inactive"
 	w_class = 3
-	req_access = list(103)
+	req_access = list(access_cent_specops)
 	health = 90
 
 	var/department = "CENTCOM"
 	var/status = 0
+	var/locked = 0
+	var/hacked = 0
```

Comment:
STATUS_ aren't actually bitflags as the status var needs to be used with a lot of switch statements. I don't know of how that works with bitflags.